### PR TITLE
disable anonymous logging in tenant helm chart

### PR DESCRIPTION
**Pull request by kanagarajkm**

### CODE REVIEW

File: helm/tenant/values.yaml [hunk 1]
```diff
@@ -162,7 +162,7 @@ tenant:
   # {"level":"INFO","errKind":"","time":"2022-04-07T21:49:33.740058549Z","message":"All MinIO sub-systems initialized successfully"}
   # Notice they are in JSON format to be consumed
   logging:
-    anonymous: true
+    anonymous: false
     json: true
     quiet: true
```
Comment (harshavardhana) [inline helm/tenant/values.yaml:167]: Even this can go

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: helm/tenant/values.yaml (hunk 1 above)
Comment (harshavardhana) [reply helm/tenant/values.yaml:167]: Simply remove it.

Comment (kanagarajkm) [reply helm/tenant/values.yaml:166]: @harshavardhana done

File: helm/tenant/values.yaml [hunk 2]
```diff
@@ -162,7 +162,6 @@ tenant:
   # {"level":"INFO","errKind":"","time":"2022-04-07T21:49:33.740058549Z","message":"All MinIO sub-systems initialized successfully"}
   # Notice they are in JSON format to be consumed
   logging:
-    anonymous: true
     json: true
     quiet: true
```
Comment (harshavardhana) [inline helm/tenant/values.yaml:166]:
```suggestion
```

File: helm/tenant/values.yaml [hunk 3]
```diff
@@ -162,7 +162,6 @@ tenant:
   # {"level":"INFO","errKind":"","time":"2022-04-07T21:49:33.740058549Z","message":"All MinIO sub-systems initialized successfully"}
   # Notice they are in JSON format to be consumed
   logging:
-    anonymous: true
     json: true
```
Comment (harshavardhana) [inline helm/tenant/values.yaml:165]:
```suggestion
```

**Review by harshavardhana — CHANGES_REQUESTED** [review]
actually we can get rid of all of them

File: helm/tenant/values.yaml (hunk 2 above)
Comment (kanagarajkm) [reply helm/tenant/values.yaml]: done

File: helm/tenant/values.yaml (hunk 3 above)
Comment (kanagarajkm) [reply helm/tenant/values.yaml]: done

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**